app/Domain/Newsletter/Controllers/NewsletterController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Newsletter\Controllers;
+
+use App\Http\Controllers\Controller;
+use Mailjet\Client;
+use Mailjet\Resources;
+use Mailjet\Response;
+
+final class NewsletterController extends Controller
+{
+    public function handle(string $email): Response
```

Comment:
Should the email address be validated?

Reply:
Maybe use the Laravel rules in this context? Something like this

```
$validator = Validator::make(
    ['email' => $email],
    ['email' => 'required|email']
);
```

Reply:
The frontend can even use the result of this validation, this is also done so for other forms.